catalog-info.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: snyk-iac-rules
+  description: 
+  labels:
+    snyk.io/businessCriticality: low
```

Comment:
How did you decide on this value? I agree it's not high but technically it is meant to be used by customers so it has some impact on them

Reply:
Hmm, that's a fair question, I didn't put a huge amount of thought into it and would be happy to revise. 

I based this mostly on surface area, e.g. it's a new feature within one single product in the overall Snyk catalog which has no direct impact on the core product flow (i.e if this disappeared or had a critical breaking issue then testing and delivery of custom rules would continue to work but creation flows would break). So the way I framed this is if this were to have a high vulnerability issue raised in a dependency what would the impact be? To me right now it's low, but happy to field opinions.

Reply:
I think your intuition was right 👍  It makes sense now that you explained it, so I'm happy with low